js/jquery.tablesorter.js

```diff
@@ -1328,6 +1328,11 @@
 				cache[ c.columns ].raw[ icell ] = tmp;
 				tmp = ts.getParsedText( c, cell, icell, tmp );
 				cache[ icell ] = tmp; // parsed
+				var tmpRow = cache[c.columns].$row;
+				if (tmpRow.length == 2) {
+				    // reapply child row
+				    $row = $row.add(tmpRow[1]);
```

Comment:
And as above, more than one child row is possible - `$row = $row.add(tmpRow);`

That "should" work, but I haven't tested it... you should be able to add all the rows back; jQuery will prevent duplication.